src/step.rs

```diff
@@ -88,5 +90,12 @@ pub(crate) enum Step {
     /// then bump the project version (depending on the rule determined from the commits) and add
     /// a new Changelog entry using the [Keep A Changelog](https://keepachangelog.com/en/1.0.0/)
     /// format.
-    UpdateProjectFromCommits,
+    UpdateProjectFromCommits {
+        #[serde(default = "default_changelog")]
+        changelog_path: String,
+    },
+}
+
+fn default_changelog() -> String {
+    "CHANGELOG.md".to_string()
```

Comment:
The problem I see here is that I am not always in the project root when running `git`, `dobby`, etc. This would cause me to accidentally place changelog files in whatever source directory I happen to have `cd`'d to.

The better default is to detect the project root, which can be done in the shell with `git rev-parse --show-toplevel`, and append `CHANGELOG.md` to that.

Reply:
Right now you can't run Dobby from anywhere other than where the config file is, so the path you provide should be relative to that file.

At some point we may have config be found at higher directories (similar to how Node searches for package.json?) but that has caused all sorts of issues in that ecosystem so it'll take a lot of thought.

Reply:
Fair point. I think doing something like using `git rev-parse --show-toplevel` is best for finding the repository root, which could be used for the default CHANGELOG location and the highest point to look for a `dobby.toml`.

If dobby cannot be run from anywhere other than the directory containing the file, and if we assume the changelog should be there or configured to be elsewhere, I think this is fine for now. I'll create a ticket for detecting the config in other directories, though.